priv/schemas/bundle_config_schema_v4.yaml

```diff
@@ -110,7 +110,9 @@ definitions:
         additionalProperties:
           type: string
       output:
-        type: string
+        type: object
```

Comment:
Also, if those are the only two properties we can have, we should drop the `additionalProperties` property.

(It seems like the `subcommands` object could use a bit more rigor, now that I look at it; that's a different PR, though)